ichnaea/content/views.py

```diff
@@ -64,13 +60,13 @@ def configure_tiles_url(asset_url):
     return (tiles_src, tiles_url)
 
 
-MAP_TILES_SRC, MAP_TILES_URL = configure_tiles_url(ASSET_URL)
+MAP_TILES_SRC, MAP_TILES_URL = configure_tiles_url(settings('asset_url'))
```

Comment:
I guess an advantage of Django-style configuration is that you can calculate these derived settings in one place, and use the same test override tools for original and derived settings. It is a little ugly to test ``MAP_TILES_URL``, but it works. 👍

Reply:
We can do derived settings using `AppConfig`, too. I'll switch that.

Reply:
I went to do that and then discovered that `MAP_TILES_URL` is more complicated than I thought it was. I'll look at addressing this when I look at #871 next.